src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,187 @@
+
+<div>
+  <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter" target="_blank" class="w-button w-button--secondary w-button--with-icon" data-icon="school">
+    Full API on MDN
+  </a>
+</div>
+
+### Summary
+- `backdrop-filter` can apply one or more filters to an element
+- The overlaying element must be at least partially transparent
+- The overlaying element will get a new stacking context
+
+{% Aside 'warning' %}
+`backdrop-filter` may harm performance.
+{% endAside %}
+
+
+## Essentials
+
+### Single Filter
+
+```css
+.blur-behind-me {
+  backdrop-filter: blur(.5rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-rgb.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/netsi1964/pen/JqBLPK" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+### Combining Filters
+
+```css
+.brighten-saturate-and-blur-behind-me {
+  backdrop-filter: brightness(150%) saturate(150%) blur(1rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-ambient_blur.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/pepf/pen/GqZkdj" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+
+## Creative Examples
+
+### Overlays
+
+```css
+.modal {
+  backdrop-filter: blur(10px);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-modal.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://mfreed7.github.io/backdrop-filter-feature/examples/scrollable.html" target="_blank">Demo</a>
+  </figcaption>
+</figure>
+
+### Text Contrast on Dynamic Backgrounds
+
+```css
+.background {
+  filter: invert(1);
+}
+
+.foreground {
+  backdrop-filter: invert(1);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-invert_color.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://tympanus.net/codrops-playground/huijing/Qqpwg5Iy/editor" target="_blank">Codrops</a>
+  </figcaption>
+</figure>
+
+### Censorship
+
+```css
+input:not(:checked) ~ .censor {
+  backdrop-filter: contrast(4) blur(20px);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-censorship.mp4" type="video/mp4">
```

Comment:
I know they're Lego figures, but I'm not super comfortable with showing a serial killer as a joke. Like, I can see people taking that as, well, insensitive. I like the idea of a joke here though. Maybe a safer (and probably equally funny) gag would be to show something that is definitely _not_ graphic? Like something that would show up on r/eyebleach, say?

Reply:
if we dont like this demo, i'll need to go find a new one. i didnt create the lego demo there, i just captured it and linked to it. not really my joke either, it's codrops. if we want a cute reveal like you describe, i'd need to go build it, host it, capture it, then swap it out here in this article. i'm not super keen on that, but understand.

Reply:
hm yeah +1 to Michael's point. Maybe a bit too dark :D

Reply:
Maybe we can omit the sensitive content overlay?

Reply:
yeah, removing it is the best idea, there's enough examples in there

Reply:
SGTM